src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java

```diff
@@ -85,8 +85,7 @@ public void init(FMLInitializationEvent event) {
 
 		// Toaster
 		if(CookingConfig.enableToaster) {
-			// TODO Toaster Recipe
-			//GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "ingotIron", 'W', "logWood", 'B', Items.water_bucket));
+			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockToaster), "   ", "IRI", "SSS", 'I', "ingotIron", 'R', "dustRedstone", 'S', "stoneSmooth"));
```

Comment:
All recipes need a more GT styled recipes. Why we using ingots why not plates, screws, springs etc?

Reply:
Because this is in CFB? You can't do that stuff without making GT a dep, and making it unusable anywhere else. That stuff goes in the coremod or scripts,

Reply:
It's easier to just do it in script, and I'm not interested in coming up with recipes for them myself.

Reply:
Mitch add some stuff for me later so i will do it myself

Reply:
Can you do that in a different PR? This is just about the toaster, and shouldn't be combined.

Reply:
sure